### ISSUE TITLE
Transfer Learning 

### ISSUE THREAD

**Issue by theRealFakeRock** [tflearn/tflearn#912]
I'm new to this so I apologize a head of time if this is the wrong way to ask this, please correct me if it is.
I have a model that I have trained, but I have decided that I want to change the output layer. Is there a way to change the output layer without completely retraining the whole model.

**Comment by iNuraj** [comment]
Yes, this is a typical application of transfer learning. Check out the lecture in here (http://cs231n.github.io/transfer-learning/) for more unsersatding of the concept.
